api/lib/domain/services/sessions-mass-import/sessions-import-validation-service.js

```diff
@@ -18,11 +18,19 @@ module.exports = {
       }
 
       if (_isSessionIdFormatValid(sessionId)) {
-        if (await _isSessionStarted({ certificationCourseRepository, sessionId })) {
+        if (await _isSessionExistingInCertificationCenter({ sessionId, certificationCenterId, sessionRepository })) {
+          if (await _isSessionStarted({ certificationCourseRepository, sessionId })) {
+            _addToErrorList({
+              errorList: sessionErrors,
+              line,
+              codes: [CERTIFICATION_SESSIONS_ERRORS.CANDIDATE_NOT_ALLOWED_FOR_STARTED_SESSION.code],
+            });
+          }
+        } else {
```

Comment:
```suggestion
        if (await !_isSessionExistingInCertificationCenter({ sessionId, certificationCenterId, sessionRepository })) {
        _addToErrorList({
            errorList: sessionErrors,
            line,
            codes: [CERTIFICATION_SESSIONS_ERRORS.SESSION_ID_NOT_EXISTING.code],
          });
        }
        if (await _isSessionStarted({ certificationCourseRepository, sessionId })) {
          _addToErrorList({
            errorList: sessionErrors,
            line,
            codes: [CERTIFICATION_SESSIONS_ERRORS.CANDIDATE_NOT_ALLOWED_FOR_STARTED_SESSION.code]
         });
        }
```

Pas forcément une suggestion vu que je ne suis pas fan du tout de devoir recourir à une négation mais ça permettrait de réduire un peu la complexité cyclomatique.

Reply:
Le soucis que je vois avec cette proposition c'est que si la session n'existe pas, on va quand même passer dans le _isSessionStarted, ce qui voudrait dire qu'elle existe.
Même si celle ci va rien retourner (parce qu'elle n'existe pas), dans la logique cette condition ne devrait même pas se vérifier